util/log.go

```diff
@@ -3,23 +3,21 @@ package util
 import (
 	"os"
 
-	logging "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/whyrusleeping/go-logging"
+	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/Sirupsen/logrus"
```

Comment:
@whyrusleeping do you want to _keep_ go-logging, or is logrus ok? (i dont care either way)